src/LiveDevelopment/LiveDevMultiBrowser.js

```diff
@@ -958,6 +959,8 @@ define(function (require, exports, module) {
 
     EventDispatcher.makeEventDispatcher(exports);
 
+    // XXXBramble: we need livePreviewAutoReload preference available to both core and extension
+
```

Comment:
It looks like your preference definition got lost somehow.  Can you add it back?

Reply:
My apologies . Must have missed that when I did `checkout` from before.